Sources/Sentry/include/SentryPerformanceTracker.h

```diff
@@ -29,6 +29,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (SentrySpanId *)startSpanWithName:(NSString *)name operation:(NSString *)operation;
 
+/**
+ * Activate the span with `spanId`
+ * to create any call to startSpan as a child.
+ * If the there is no span with the fiven spanId
+ * block is executed anyway.
+ *
+ * @param spanId Id of the span to activate
+ * @param block Block to invoke while span is active
+ */
+- (void)activateSpan:(SentrySpanId *)spanId duringBlock:(void (^)(void))block;
```

Comment:
`m`: I think it would be better to only expose `measureSpanWithDescription` or something and hide `activeSpan`. When interacting with the SentryPerformanceTracker I don't want to know that there are active spans.

Reply:
`measureSpanWithDescription` creates a new span. But sometime we need to add new spans to a previously created transaction.